src/platforms/linux/linuxcryptosettings.cpp

```diff
@@ -46,108 +55,54 @@ LinuxCryptoSettings::LinuxCryptoSettings() {
     return;
   }
 
-  // Check if "org.freedesktop.Accounts" "AutomaticLoginUsers" is empty.
-  QDBusInterface iface("org.freedesktop.Accounts", "/org/freedesktop/Accounts",
-                       "org.freedesktop.DBus.Properties",
-                       QDBusConnection::systemBus());
-
-  QDBusMessage autoLoginReply =
-      iface.call("Get", "org.freedesktop.Accounts", "AutomaticLoginUsers");
-  if (autoLoginReply.type() != QDBusMessage::ReplyMessage) {
-    logger.info() << "Encrypted settings with libsecrets is not supported "
-                     "(incompatible reply)";
+  /* Try to lookup the encryption key. This can fail for a variety of reasons.
+   * It's also not reliable, which is why we want to move users onto the XDG
+   * Secrets portal for managing their encrypted settings.
+   */
+  GError* err = nullptr;
+  auto schema = cryptosettings_get_schema();
+  gchar* password = secret_password_lookup_sync(schema, nullptr, &err, nullptr);
+  if (err != nullptr) {
+    g_error_free(err);
```

Comment:
Do we want to log the error in this block, as we do currently?

Reply:
Sure, I can put the error message back.